extensions/guacamole-auth-ldap/src/main/java/org/glyptodon/guacamole/auth/ldap/connection/ConnectionService.java

```diff
@@ -109,11 +109,38 @@
             // possibly be null
             assert(userDN != null);
 
-            // Find all Guacamole connections for the given user
+            // If group base DN is specified search for user groups
+            String groupBaseDN = confService.getGroupBaseDN();
+            // Create a separate filter for user groups
```

Comment:
While you're fixing the indentation, mind maybe adding a blank line above here? This looks rather jumbled as-is. Everything is just squished together.